resources/assets/lib/beatmapsets/beatmapset-json.ts

```diff
@@ -13,6 +13,17 @@ interface BeatmapsetCovers {
   slimcover: string;
 }
 
+export function isBeatmapsetNominationEvent(x: BeatmapsetEvent): x is BeatmapsetNominationEvent {
+  return x.type === 'nominate' && x.comment != null && Array.isArray(x.comment.modes);
```

Comment:
```suggestion
  return x.type === 'nominate' && Array.isArray(x.comment?.modes);
```

Reply:
But that will call `isArray` on `undefined` :thinking:

Reply:
which returns false anyway?